requirements-tests.txt

```diff
@@ -5,3 +5,4 @@ flaky
 pytest
 pytest-cov
 pytest-timeout
+pathlib
```

Comment:
```suggestion
pathlib2; python_version < '3.4'
```

Reply:
@brant-ruan do you think this is needed if you remove the `pathlib` part you added in the test?

Reply:
> @brant-ruan do you think this is needed if you remove the `pathlib` part you added in the test?

oh yes you're right. There's no need importing pathlib, just to touch a file!